Ontologies.Mappings/src/Mappings/v1/Willow/mapped_v1_dtdlv2_Willow.json

```diff
@@ -70,10 +82,26 @@
       "InputDtmi": "dtmi:mapped:core:Air_Flow_Status;1",
       "OutputDtmi": "dtmi:com:willowinc:State;1"
     },
+    {
+      "InputDtmi": "dtmi:mapped:core:Air_Quality_Sensing_Device;1",
+      "OutputDtmi": "dtmi:com:willowinc:Device;1"
```

Comment:
Device seems rather generic. We have dtmi:com:willowinc:SensorEquipment;1 or what about dtmi:com:willowinc:IAQSensorEquipment;1 ?